CDCgov/prime-reportstream
# 13598 address database varchar size limits

This PR extends the length of the `report_file`.`schema_name` column so longer strings can be stored and removes temporary code that truncates the data to be stored in this column. It additionally extends the length of many other varchar columns throughout the database.

Test Steps:
1. Run `clean` followed by `package` to apply db migrations and run tests.
2. Confirm the database has been updated by running the following query and verifying the value is null:
```
SELECT character_maximum_length 
FROM information_schema.columns 
WHERE table_schema = 'public' 
AND table_name  = 'report_file' 
AND column_name  = 'schema_name';
```

## Changes
- Add migration to alter size of `report_file`.`schema_name` and many other varchar database columns
- Remove temporary code to truncate schema name prior to inserting to database
- Fix actionHistory to correctly handle varchar columns without a size limit
- Add and update tests

## Checklist

### Testing

## Linked Issues
- #13598

## To be done
- #14721
